src/org/zaproxy/zap/extension/invoke/resources/help/contents/options.html

```diff
@@ -54,6 +54,16 @@ <H3>Parameters</H3>
 			<td>%postdata%</td>
 			<td>the POST data sent, if any, with any newlines replaced with "\n"</td>
 		</tr>
+		<tr>
+			<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
+			<td>%msgid%</td>
+			<td>the HTTP message id to fetch data the API (/JSON/core/view/message/?id=$msgid), does not alway exist</td>

```

Comment:
IMO this still needs tweaks (missing a "from"?).

Reply:
‘The HTTP message id to fetch from the API....., does not always exist.’

I’m working on a 4” screen right now. If it isn’t found is a default returned? (0 or -1)

Reply:
Yes, returns -1.

Reply:
Might want to mention that as well.